.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "printWidth": 120,
+    "tabWidth": 4,
+    "semi": false,
+    "arrowParens": "always",
+    "trailingComma": "none"
```

Comment:
Let's use the default "es5", since it allows trailing commas. Trailing commas are useful in keeping code diffs limited to the exact lines added/removed.